modules/configuring-cluster-monitoring.adoc

```diff
@@ -52,6 +52,10 @@ data:
       baseImage: quay.io/coreos/kube-state-metrics
       nodeSelector:
         node-role.kubernetes.io/infra: ""
+    openshiftStateMetrics:
+      baseImage: openshift/openshift-state-metrics
```

Comment:
1. we want to increase the storage capacity for Prometheus, so there is not need to mention components except prometheusK8s and alertmanagerMain
2. baseImage is not needed for each component
3. there is not setting for `auth`
```
    auth:
      baseImage: openshift/oauth-proxy
```
component see from https://docs.openshift.com/container-platform/4.6/monitoring/configuring-the-monitoring-stack.html#configurable-monitoring-components_configuring-the-monitoring-stack
note: configuration for thanosQuerier is since 4.5
4. ```
    prometheusOperator:
      baseImage: quay.io/coreos/prometheus-operator
      prometheusConfigReloaderBaseImage: quay.io/coreos/prometheus-config-reloader
      configReloaderBaseImage: quay.io/coreos/configmap-reload
```
these settings should be removed

Reply:
@duritong PTAL at juzhao's comments. Thanks!

Reply:
@duritong PTAL at juzhao's comments. Thank you.